openmetadata-ui/src/main/resources/ui/src/components/Explore/ColumnSummary/ColumnSummary.component.tsx

```diff
@@ -0,0 +1,67 @@
+import { Divider, Space, Typography } from 'antd';
+import { toLower } from 'lodash';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { getTagValue } from '../../../utils/CommonUtils';
+import SVGIcons from '../../../utils/SvgUtils';
+import TagsViewer from '../../tags-viewer/tags-viewer';
+import { BasicColumnInfo, ColumnSummaryProps } from './ColumnSummary.interface';
+
+const { Text, Paragraph } = Typography;
+
+export default function ColumnSummary({ columns }: ColumnSummaryProps) {
+  const { t } = useTranslation();
+
+  const formattedColumnsData: BasicColumnInfo[] =
+    columns &&
+    columns.map((column) => ({
+      name: column.name,
+      type: column.dataType,
+      tags: column.tags,
+      description: column.description,
+    }));
```

Comment:
wrap it with useMemo to avoid multiple executions

Reply:
Done